gtfs/__main__.py

```diff
@@ -39,50 +42,76 @@ def check_bbox(bbox: str) -> Bbox:
 @app.command()
 def list_feeds(
     bbox: Annotated[
-        str,
+        Union[str, None],
         typer.Option(
             "--bbox",
             "-b",
             help="pass value as a string separated by commas like this: min_x,min_y,max_x,max_y",
             callback=check_bbox,
         ),
-    ],
+    ] = None,
     predicate: Annotated[
-        Predicate,
+        Union[Predicate, None],
         typer.Option(
             "--predicate",
-            "-p",
+            "-pd",
             help="the gtfs feed should intersect or should be contained inside the user's bbox",
         ),
-    ] = Predicate.intersects,
+    ] = None,
+    pretty: Annotated[
+        bool,
+        typer.Option(
+            "--pretty",
+            "-pt",
+            help="display feeds inside a pretty table",
+        ),
+    ] = False,
 ) -> None:
     """Filter feeds spatially based on bounding box."""
-    spinner("Fetching feeds...", 1)
-    ptable = ColorTable(["Feed Source", "Transit URL", "Bounding Box"], theme=Themes.OCEAN)
-    for src in feed_sources:
-        feed_bbox: Bbox = src.bbox
-        if predicate == "contains":
-            if not bbox_contains_bbox(feed_bbox, bbox):
-                continue
-        elif predicate == "intersects":
-            if (not bbox_intersects_bbox(feed_bbox, bbox)) and (
-                not bbox_intersects_bbox(bbox, feed_bbox)
-            ):
-                continue
-
-        row = [
-            src.__name__,
-            src.url,
-            [feed_bbox.min_x, feed_bbox.min_y, feed_bbox.max_x, feed_bbox.max_y],
-        ]
-        ptable.add_row(row)
-
-    print(
-        "\n" + f"Feeds based on bbox input {bbox} and "
-        f"for predicate={predicate.value} are as follows:"
-    )
-    print("\n" + ptable.get_string())
-    console.print("All done!", style=success)
+    if bbox is None and predicate is not None:
+        raise typer.BadParameter(
+            f"Please pass a bbox if you want to filter feeds spatially based on predicate = {predicate}!"
+        )
+    elif bbox is not None and predicate is None:
+        raise typer.BadParameter(
+            f"Please pass a predicate if you want to filter feeds spatially based on bbox = {bbox}!"
+        )
+    else:
+        spinner("Fetching feeds...", 1)
+        typer.secho("Filtered feeds are:", fg=typer.colors.BLUE)
```

Comment:
Same for this line, I think it's better to have a clean output, either just the table or just the newline separated names of the gtfs feeds.